src/Microsoft.Identity.Client/Platforms/net45/WindowsFormsWebAuthenticationDialogBase.cs

```diff
@@ -292,45 +292,39 @@ private void InitializeComponent()
                 : Screen.PrimaryScreen;
 
             // Window height is set to 70% of the screen height.
-            int uiHeight = (int) (Math.Max(screen.WorkingArea.Height, 160)*70.0/DpiHelper.ZoomPercent);
-            webBrowserPanel = new Panel();
-            webBrowserPanel.SuspendLayout();
+            int uiHeight = (int)(Math.Max(screen.WorkingArea.Height, 160) * 70.0 / DpiHelper.ZoomPercent);
+            _webBrowserPanel = new Panel();
+            _webBrowserPanel.SuspendLayout();
             SuspendLayout();
 
-            // 
             // webBrowser
-            // 
-            webBrowser.Dock = DockStyle.Fill;
-            webBrowser.Location = new Point(0, 25);
-            webBrowser.MinimumSize = new Size(20, 20);
-            webBrowser.Name = "webBrowser";
-            webBrowser.Size = new Size(UIWidth, 565);
-            webBrowser.TabIndex = 1;
-            webBrowser.IsWebBrowserContextMenuEnabled = false;
-
-            // 
+            _webBrowser.Dock = DockStyle.Fill;
+            _webBrowser.Location = new Point(0, 25);
+            _webBrowser.MinimumSize = new Size(20, 20);
+            _webBrowser.Name = "_webBrowser";
```

Comment:
The variable name of the field was originally webBrowser (the second 'b' is capital).  I think names are generally either camelCase or PascalCase, so webbrowser in all lowercase seems wrong.  I was following the [CoreFX](https://github.com/dotnet/corefx) [C# Coding Style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md) guidelines in renaming the private field.  Does this project have a different coding style guide?

> We use `_camelCase` for internal and private fields. Prefix instance fields with `_`.